src/pages/search-page/tabs/components/tab/tab.component.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+
+import SC from './styled';
+
+interface Props {
+  active: boolean;
+  tabLink: any;
+  label: string;
+}
+
+export const Tab: FC<Props> = ({ active, tabLink, label, children }) => (
```

Comment:
Bruk heller `FC<PropsWithChildren<Props>>` for at TypeScript ikke skal klage der man bruker komponenten.

Akkurat nå er den i bruk i en `.jsx` fil, som gjør at vi ikke ser noen feilmeldinger fra kompilator.

Reply:
fixed